casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala

```diff
@@ -142,10 +142,16 @@ object Genesis {
     )
 
   private def fromLine(line: String): Either[String, Vault] = line.split(",") match {
-    case Array(ethAddress, initRevBalanceStr, _) =>
+    case Array(ethAddressString, initRevBalanceStr, _) =>
       Try(initRevBalanceStr.toLong) match {
         case Success(initRevBalance) =>
-          Right(Vault(RevAddress.fromEthAddress(ethAddress), initRevBalance))
+          RevAddress.fromEthAddress(ethAddressString) match {
```

Comment:
This can be replaced by 
```scala
RevAddress.fromEthAddress(ethAddressString)
.map(Vault(_, initRevBalance))
.asRight(s"Ethereum address $ethAddressString is invalid.")
```

Reply:
Done